PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -633,6 +635,15 @@ void ScintillaEditView::setJsonLexer()
 	execute(SCI_SETPROPERTY, reinterpret_cast<WPARAM>("fold.preprocessor"), reinterpret_cast<LPARAM>("1"));
 }
 
+void ScintillaEditView::setCssLexer(LangType type) 
+{
+	setLexer(SCLEX_CSS, L_CSS, LIST_0 | LIST_1);
+	if (type == L_LESS) 

```

Comment:
Should this just be `if (type == L_LESS || type == L_SCSS)` or should they be setting different properties for each?